proto/api/v1/delegated_registry_config_service.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+option go_package = "v1";
+
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "api/v1/empty.proto";
+import "storage/delegated_registry_config.proto";
```

Comment:
I highly recommend not including the storage proto in the api proto.  That introduces a handcuff to updating the storage later.

Reply:
I have heard that in conversation - wasn't sure if that was a 'big bang' plan or just going forward, I'll swap it out for something specific for the API

Reply:
👍

Reply:
Believe this is addressed in the latest commit(s), let me know if thats no what you had in mind.